regcore/migrations/0002_preamble.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this migration be re-numbered to account for the migrations from #38? Not sure if this matters.